package.json

```diff
@@ -11,7 +11,8 @@
     "build": "bit-docs -df",
     "rebuild-assets": "bit-docs -dfs",
     "clean": "rm -rf academy",
-    "deploy": "node scripts/hubspot-deploy"
+    "deploy": "node scripts/hubspot-deploy",
+    "serve": "./node_modules/.bin/http-server ./academy -p 3000 -c -1"
```

Comment:
I’m assuming the `./academy` bit will make it serve that folder as the route, is that right?

I prefer things as they are because then the URL structure matches what’s deployed to Bitovi.com (with `/academy/`), which makes it easier to copy & paste URLs and replace the root domain.

I also prefer not having a hard-coded port so there can’t be conflicts with other projects.

If you do merge these changes in, I would consider making this the `start` script so we are using a standard npm script name. _Maybe_ even run the `dev` script with this so it’s just one command to start both.

Reply:
I think it1s `./academy` cause that's what bit-docs uses as a `dist` folder for this project. I don't think it's in the URL. Agree that it should be if that's easy to facilitate.

I would agree with integrating this with `dev` if possible.

Note: you don't need the `./node_modules/.bin/`. Anything in that folder is automatically in the path when running npm scripts. It can just be `"serve": "http-server ./academy -p 3000 -c -1"`

Reply:
- Server now serves from the project root.
- Changed the script name to `start`.

@chasenlehara if a port is not specified the default is 8080. It's been a long time since I've worked on a project where the port was not specified either through the command line or a configuration file - but I'm open to suggestions for other methods. I changed it to 4444 which is not as common.

Simply concatenating the two commands to run serially doesn't work (they both hold the main thread). I considered running the first one in the background on another process but that means it can't be killed by "ctrl+c" and it's "hidden" from the dev. So I opted for keeping them separate and noting in the readme how to run both in separate terminals - again open to other suggestions.

Reply:
Ok, sounds good. I still personally don’t agree with setting a port (`http-server` defaults with `8080` and will increment until it finds an open port, so I think we should let it do that and not set our own port), but no worries, I can still use `npx http-server` on my own and not this script.

Reply:
> So I opted for keeping them separate and noting in the readme how to run both in separate terminals - again open to other suggestions.

Folks on hatchify keep telling me there's a way to have a single command run two different processes cleanly.  Haven't seen it.  @nlundquist did you tell me about this?

Reply:
Yeah, the package to do this most easily is [concurrently](https://www.npmjs.com/package/concurrently). It works quite well to multiplex two different terminal programs together, and all you need to do is tweak your `npm run dev` to be something like: 

`concurrently "http-server ./academy -p 3000 -c -1" "bit-docs -dw"`

this could be shortened to something like:

`concurrently: "npm:dev:bit-docs" "npm:dev:http-server"`

if you added "dev:bit-docs" and "dev:http-server" to the `package.json` scripts

Reply:
Removed port.